implementations/rust/ockam/ockam/src/forwarder.rs

```diff
@@ -70,6 +70,7 @@ impl Forwarder {
             forward_route,
             payload: Some(registration_payload.clone()),
         };
+        let _ = ctx.stop_worker(address.clone()).await;
```

Comment:
Shouldn't we at least add prefix to the provided address, as in elixir? Not doing that looks bad even for demo purposes...
@mrinalwadhwa

Reply:
Sorry, this was a leftover from a previous revision. While it does not affect naming it is not relevant for this PR.